python/Ganga/test/GPI/FileTests/TestMassStorageWN.py

```diff
@@ -0,0 +1,249 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+import tempfile
+import datetime
+import time
+import random
+import os
+import shutil
+import copy
+import string
+
+def generateUniqueTempFile( ext = '.txt' ):
```

Comment:
If people are happy for me to move stuff there. I was planning on breaking this out into a common file in this dir but it felt overkill for what is about ~10 lines of code.